### PR TITLE
Create compare_number.cpp

### PR DESCRIPTION
**Pull request by Ericostco**
The first version in c++

### CODE REVIEW

File: compare_number.cpp [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#include<iostream>
+using namespace std;
+
+int main(){
+    int num1=0,num2 = 0, num3 = 0; //initailize variables
+    cout<<"please input 3 number" << endl;
+    cin >> num1 >> num2>>num3;
```
Comment (tianyo1110) [inline compare_number.cpp:7]: cin >> num1 >> num2 >> num3;

File: compare_number.cpp [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#include<iostream>
+using namespace std;
+
+int main(){
+    int num1=0,num2 = 0, num3 = 0; //initailize variables
```
Comment (tianyo1110) [inline compare_number.cpp:5]: int num1 = 0 , num2 = 0 , num3 = 0;

**Review by tianyo1110 — CHANGES_REQUESTED** [review]

File: compare_number.cpp [hunk 3]
```diff
@@ -2,7 +2,7 @@
 using namespace std;
 
 int main(){
-    int num1=0,num2 = 0, num3 = 0; //initailize variables
+    int num1=0, num2 = 0, num3 = 0; //initailize variables
```
Comment (qaz5823091) [inline compare_number.cpp:5]: Great!

**Review by qaz5823091 — APPROVED** [review]
Great!

**Review by tianyo1110 — APPROVED** [review]
OKOK

**Pull request merged**